src/database/seeders/20210710124704-add_students_groupId.js

```diff
@@ -70,7 +70,7 @@ module.exports = {
       ]);
     });
   },
-  down: async (queryInterface, Sequelize) => {
+  down: async (queryInterface) => {
     return await queryInterface.bulkDelete('Students', null);
```

Comment:
Эти изменения здесь не нужны
```suggestion
 down: async (queryInterface, Sequelize) => {=> {
    return await queryInterface.bulkDelete('Students', null);
```

Reply:
@OmegonAlpharius  Вернул обратно